src/App.css

```diff
@@ -1,38 +1,261 @@
+
+html { box-sizing: border-box; font-size:125%;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+img { width: 100%; display: block; height: auto;}
+
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap");
+@import url("https://fonts.cdnfonts.com/css/pokemon-solid");
+
+*{
+    padding: 0;
+    margin: 0;
+}
+
+body{
+    background-color: rgb(213, 0, 0);
+    font-family: 'Open Sans', sans-serif;
+    color: yellow;
+    text-shadow: 5px 5px 0px #00077C;
+}
+
+.title{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-family: 'Pokemon Solid', sans-serif;
+    color: yellow;
+    text-shadow: 5px 5px 0px #00077C;
+    font-size: 2rem;
+    flex-direction: column;
+}
+
+.title form{
+    font-size: 1rem;
+}
+
+.container{
+    width: 90%;
+    margin: auto;
+    padding-top: 50px;
+    display: flex;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.left-content,
+.right-content{
+    flex-basis: 50%;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+.left-content{
+    display: flex;
+    flex-wrap: wrap;
+    width: 50%;
+    justify-content: space-evenly;
+    gap: 1rem;
+}
+
+.right-content{
+    width: 50%;
+    text-align: center;
+    color: black;
+    position: fixed;
+    right: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 60%;
   }
+
+.right-content img{
+    width: 30%;
+}
+
+.display{
+    width: 45%;
+    background: rgba(255, 255, 255, 0.2);
+    border-radius: 16px;
+    box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+    backdrop-filter: blur(5px);
+    -webkit-backdrop-filter: blur(5px);
+    border: 1px solid rgba(255, 255, 255, 0.3);
+    border-radius: 10px;
+    box-shadow: 0 5px 5px rgba(0,0,0,0.5);
+    padding: 0 1.5rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-sizing: border-box;
+}
+
+.display h2{
```

Comment:
Desktop screen size content overflows out of container.

![Screen Shot 2023-03-29 at 2 48 19 PM](https://user-images.githubusercontent.com/109436128/228638517-6a0a1145-a274-42a2-82b9-0ccfbbe55603.png)